esmvaltool/cmor/_fixes/CMIP6/BCC_CSM2_MR.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
```

Comment:
Please don't disable pylint: these are valid complaints, disabling them makes it more difficult to assess the quality of the various parts of our codebase.

Reply:
Removed all the pylint comments.